packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -337,12 +338,14 @@ export default class PerformanceMonitoring {
         }
 
         let outcome
-        if (data.status != 'abort') {
+        if (data.status != 'abort' && !data.aborted) {
           if (status >= 400 || status == 0) {
             outcome = OUTCOME_FAILURE
           } else {
             outcome = OUTCOME_SUCCESS
           }
+        } else {
+          outcome = OUTCOME_UNKNOWN
```

Comment:
We should not need to set this, APM server treats the outcome as `Unknown` if its not set. 

Could you double check this one?

Reply:
You are right, transaction.outcome is set to `Unknown` if it's not set. Nevertheless,  with the span.outcome it's not occurring the same.

Screenshot of aborted span request **without** the change:

<img width="660" alt="Screenshot 2022-04-28 at 12 52 58" src="https://user-images.githubusercontent.com/15065076/165737015-a20cd2f0-5526-4dcd-b52e-85c3e17c55b3.png">

Screenshot of aborted span request **with** the change:

<img width="652" alt="Screenshot 2022-04-28 at 12 53 41" src="https://user-images.githubusercontent.com/15065076/165737144-7dc16309-ae09-4c7c-a019-e2a276cd9e4b.png">

Reply:
Wow TIL, not sure if it's the intended behavior but all good from our side 👍🏽